### PR TITLE
fix: pseudo scroll locker styles fix

### PR DESCRIPTION
**Pull request by ala-n**

### CODE REVIEW

**Review by Sisha3342 — APPROVED** [review]

**Review by yadamskaya — APPROVED** [review]

File: src/modules/esl-utils/dom/scroll.less [hunk 1]
```diff
@@ -40,10 +40,12 @@ html.esl-disable-scroll {
       right: 0;
       bottom: 0;
       overflow-y: scroll;
+      // Pseudo scroll should be visible above the content
+      z-index: 100000;
```
Comment (dshovchko) [inline src/modules/esl-utils/dom/scroll.less:44]: I've seen some have great values. For example, AddThis has from 100000 and above.

**Review by dshovchko — APPROVED** [review]

File: src/modules/esl-utils/dom/scroll.less (hunk 1 above)
Comment (ala-n) [reply src/modules/esl-utils/dom/scroll.less]: Let's go with a little compromise, as you suggested: 999999 :) Anyway, that's just a pseudo scroll and just a CSS that is overridable by consumers.
Thanks!

**Review by julia-murashko — APPROVED** [review]

**Pull request merged**